docs/dev-tools/zombienet/overview.md

```diff
@@ -133,99 +135,102 @@ provider = "<provider>"
 ...
 ```
 
+At the moment, Zombienet supports the following providers: `kubernetes`, `podman`, and `native`.
```

Comment:
Yes, that makes sense. I kept using the word `native` in some cases because it is the keyword recognized by Zombienet, but adding a small clarification like those parentheses would make it clearer 🚀